group_vars/bm_wrk_win.yml

```diff
@@ -1,3 +1,12 @@
-ov_osdp: 'WindowsServer2016_singleframe_v1.0.1'           # Name of the OneView Server Profile Template to use for worker nodes
-ov_template: 'Windows Worker Node (single-frame) v1.0.1'  # Name of the OS Deployment Plan, we should probably be able to figure it out
-ov_ansible_connection_name: 'ansible'                     # Name of the connection in the Server Profile that connects to the ansible network
+ansible_user: '{{ win_username }}'
```

Comment:
most of the variables (such as ansible_*) belong to windows_box.yml. They apply to all windows boxes and not just BM windows machines

Reply:
I agree we should be consistent about where the variables are stored.  I've removed all the ansible_ variables from the bm_wrk_win.yml and vm_wrk_win.yml files and moved them to windows_box.yml since they are the same for all Windows systems.  Question now is do we keep the Windows-specific variables in group_vars/all or move all Windows-specific variables to the windows_box.yml file?